nabo/nabo.h

```diff
@@ -206,24 +205,24 @@ libnabo differs from \ref ANN on the following points:
 //! Namespace for Nabo
 namespace Nabo
 {
-	//! \defgroup public public interface 
+	//! \defgroup public public interface
 	//@{
-	
+
 	//! version of the Nabo library as string
 	#define NABO_VERSION "1.0.6"
 	//! version of the Nabo library as an int
 	#define NABO_VERSION_INT 10006
-	
+
 	//! Parameter vector
-	struct Parameters: public std::map<std::string, boost::any>
+	struct Parameters: public std::map<std::string, double>
 	{
 		//! Create an empty parameter vector
 		Parameters(){}
 		//! Create a parameter vector with a single entry
 		/** \param key entry key
 		 * \param value entry value
 		 */
-		Parameters(const std::string& key, const boost::any& value){(*this)[key] = value;}
+		Parameters(const std::string& key, double value){(*this)[key] = value;}
```

Comment:
I'm not sure whether `double` is a good enough replacement at the moment. Maybe it is at least until `std::any` is finally available for this library (c++17).
@stephanemagnenat maybe you should decide that?